webpack/webpack.config.dev-server.js

```diff
@@ -9,14 +9,14 @@ var commonLoaders = [
      * Read more http://babeljs.io/docs/usage/experimental/
      */
     test: /\.js$|\.jsx$/,
-    loader: 'babel',
+    loader: 'babel-loader',
     // Reason why we put this here instead of babelrc
     // https://github.com/gaearon/react-transform-hmr/issues/5#issuecomment-142313637
     query: {
       "presets": ["es2015", "react", "stage-0"]
     },
     include: path.join(__dirname, '..', 'app'),
-    exclude: path.join(__dirname, '/node_modules/')
+    exclude: path.join(__dirname, '..', 'node_modules')
```

Comment:
I was also wondering if this was an issue or if I was missing something... I see I was right, nice catch :+1: